### PR TITLE
ui-revamp

### LINKED ISSUE #367: ✨ Improvement: Implement Figma design
### 📜 Description

https://www.figma.com/design/0Acu1rj33FPBuCzFnqDiyW/100xdevs-Working?node-id=7251-100257&node-type=section&t=SAe4yVIH8cO5ZSow-0

Implement this design

image

### 👟 Reproduction steps

go to jobs page and job page

### 👍 Expected behavior

_No response_

### 📃 Provide any additional context for the Bug.

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/code100x/job-board/blob/main/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Ashutoshpadhi629**
fixes #367 
- new UI for jobs route

https://github.com/user-attachments/assets/c461b1d2-bfb8-41d4-8f24-8096b725ade5

https://github.com/user-attachments/assets/b34d42a9-f90f-4479-a48b-04baab39ea59

### CODE REVIEW

**Comment by Ashutoshpadhi629** [comment]
@VineeTagarwaL-code  did almost all the changes from the Figma file, there are still a few more changes:-
- navbar and footer
- filter logic for Employment Type
- Tags for Jobs schema

and a few more small improvements. Happy to do these all here.

**Comment by VineeTagarwaL-code** [comment]
have you worked on the footer too? cant see it getting changed

**Comment by Ashutoshpadhi629** [comment]
@VineeTagarwaL-code I haven't worked on footer and navbar . Should I do that here?

**Comment by VineeTagarwaL-code** [comment]
when are you completing this >?

**Comment by Ashutoshpadhi629** [comment]
@VineeTagarwaL-code almost done, will commit soon

**Comment by Ashutoshpadhi629** [comment]
@VineeTagarwaL-code , Navbar and Footer is Completed

https://github.com/user-attachments/assets/39adff3d-2442-4bba-98d2-47513a33a2e7


https://github.com/user-attachments/assets/a8a6e6be-ceb2-4c41-ad40-a242cbe540e7

**Comment by Ashutoshpadhi629** [comment]
@VineeTagarwaL-code, I need to change the job prisma schema to add skills in the job card , and the create job form is not responsive , should i work on them in here,

Screenshot 2024-09-21 at 4 42 25 PM

**Comment by VineeTagarwaL-code** [comment]
yeah do

**Comment by Ashutoshpadhi629** [comment]
hey @VineeTagarwaL-code ,  heres a live deployment link [https://job-board-git-ui-revamp-ashutosh-padhis-projects.vercel.app/](url) ,
- changed the Prisma schema and seed.ts
- create job form is now responsive
- pixel perfect ui from figma

**Comment by VineeTagarwaL-code** [comment]
looks clean and good, reviewing the code right now

File: src/components/all-jobs.tsx [hunk 1]
```diff
@@ -30,32 +33,64 @@ const AllJobs = async ({ searchParams }: PaginatorProps) => {
...
+              key={job.id}
+              href={`/jobs/${job.id}`}
+              className="sm:text-sm text-xs text-slate-500 dark:text-slate-400 font-medium flex flex-col border p-6 bg-slate-100 gap-4 dark:bg-slate-900 hover:bg-slate-200 dark:hover:bg-slate-800 rounded-lg "
+            >
+              <div className="flex w-full gap-3">
+                <div>
+                  {/* Todo : replace with job.companyLogo */}
+                  <Image
+                    alt="company_logo"
+                    src={'/spotify.png'}
```
Comment (VineeTagarwaL-code) [inline src/components/all-jobs.tsx:46]: why the src is harcoded here ? while seeing put the image link / logo

**Review by VineeTagarwaL-code — CHANGES_REQUESTED** [review]

File: src/components/all-jobs.tsx (hunk 1 above)
Comment (Ashutoshpadhi629) [reply src/components/all-jobs.tsx:46]: for now, we don't have the company image in the seed file, so I hardcoded it

**Comment by Ashutoshpadhi629** [comment]
Hey @VineeTagarwaL-code , I resolved all the conflicts

**Comment by VineeTagarwaL-code** [comment]
fix the conflict will merge after that

**Comment by Ashutoshpadhi629** [comment]
@VineeTagarwaL-code , resolved the conflicts

**Pull request merged**